generator/templates/scss/src/plugins/bootstrap-vue.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+
+// import BootstrapVue from 'bootstrap-vue'
+import { ButtonPlugin } from 'bootstrap-vue'
```

Comment:
Was this for testing purposes? I imagine we would want to include the whole library by default.

Reply:
I think it's better to import all plugins independently and let the user remove, or comment, the plugins unused.
If the user wants to use the whole library, can use the default mode which includes all css and plugins/components, but without the possibility to customize the variables and the bootstrap build

Reply:
I think all of the addressed changes make sense. The only one I think we should change is this.  The CLI plugin is used by people of varying experience levels with vue/webpack/etc.  I think we should probably import the whole bootstrap-vue library by default, with the option to change to independent imports (basically the opposite of what is here).  

If you feel really strongly about this go ahead and mark the comment as resolved and I'll merge it.